### PR TITLE
Adds yi to vox syllables

### PR DESCRIPTION
**Pull request by Intigracy**
removes partial pathing in species.dm

adds yi to syllables

List of all current syllables:
cha
chi
ha
hi
ka
kah
ki
ta
ti
ya
ya
yi

### CODE REVIEW

**Comment by clusterfack** [comment]
Why though

**Comment by Intigracy** [comment]
updoot

File: code/names.dm [hunk 1]
```diff
@@ -15,4 +15,4 @@ var/list/adjectives = file2list("config/names/adjectives.txt")
 //loaded on startup because of "
 //would include in rsc if ' was used
 
-var/list/vox_name_syllables = list("ti","hi","ki","ya","ta","ha","ka","ya","chi","cha","kah")
\ No newline at end of file
+var/list/vox_name_syllables = list("cha","chi","ha","hi","ka","kah","ki","ta","ti","ya","yi")
```
Comment (clusterfack) [inline code/names.dm]: Can you believe this nigga removed the second ya? HARAM

**Comment by gbasood** [comment]
But the second ya is silent! :^)

**Comment by Intigracy** [comment]
Re-implemented much sought after double ya

**Pull request merged**